nemo/collections/asr/parts/submodules/rnnt_loop_labels_computer.py

```diff
@@ -498,6 +511,24 @@ def loop_labels_cuda_graphs(
         self.state.encoder_output_length[: encoder_output_length.shape[0]].copy_(encoder_output_length)
         # set length to zero for elements outside the current batch
         self.state.encoder_output_length[current_batch_size:].fill_(0)
+
+        if prev_state is None:
+            # initial state
+            self.decoder.batch_replace_states_all(
+                src_states=self.decoder.initialize_state(self.state.encoder_output_projected),
+                dst_states=self.state.decoder_state,
+            )
+        else:
+            self.decoder.batch_replace_states_all(
+                src_states=prev_state, dst_states=self.state.decoder_state,
+            )
+
+        if prev_labels is None:
```

Comment:
In an ideal implementation, you would be able to support "continuous batching". What I mean is, if a single element finishes before the others in the batch, then a new element will be placed into the batch. I realize that NeMo isn't ready for this yet.

Reply:
`prev_labels` is a batch of the "last labels from the previous decoding iteration".
If you want to start a new sequence ("continuous batching"), just place `<SOS>` symbol for the corresponding item (=blank for all our models, `blank_as_pad=True`). This is already supported in this PR, see 
https://github.com/NVIDIA/NeMo/pull/9106/files#diff-c695cda936f43469cd972499fb0bf557b5abbd852395e2c5f52faae37c95f4eeR752

We also need to make zero initial state for this item in the batch, I'm also planning to implement this.